components/d2l-applied-filters/d2l-applied-filters.js

```diff
@@ -60,9 +61,22 @@ class D2lAppliedFilters extends RtlMixin(LocalizeStaticMixin(LitElement)) {
 				color: var(--d2l-color-corundum);
 			}
 
+			#d2l-clear-filters-button {
+				margin-left: 3px;
+				margin-right: 3px;
+			}
+
+			#d2l-list-holder {
+				flex: 1;
+			}
+
 			:host([hidden]) {
 				display: none;
 			}
+
+			[hidden] {
+				display: none !important;
```

Comment:
What does !important mean?

Reply:
It means that other CSS can't override `display: none`. Adding this line was recommended to me by Jeff as one of the rare cases where `!important` is not unacceptable.

This doc

Reply:
It means that other CSS can't override `display: none`. Adding this line was recommended to me by Jeff as one of the rare cases where `!important` is not unacceptable. It's acceptable in this case because we are hiding a component and we really don't want another part of the page deciding that it should be visible. `!important` is generally a bad practice because it voilates the natural cascading in your stylesheets. I think of it kind of like `GOTO` but for CSS.

@jstefaniuk-d2l can let me know if I have accurately explained the reasoning.

This doc explains it under the "The !important exception"
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

Reply:
It disregards concepts like specificity to ensure this declaration is always the one used when an element has the `hidden` attribute.

Reply:
Which element is this selector being applied to?  Is it the subtle button?  The subtle button already contains host styles that honor the `hidden` attribute that is being applied.  In this case, the component will be scoped by shadowDOM so there isn't really a major concern of random things on the page overriding in this scope.

It is true that we try to avoid using `!important` for the specificity reason.  It basically trumps everything else, even inline styles.  It's basically the nuclear option.  Using it really tends to put us in a tight spot if we need to override the property.  For that reason, we only use it when it is absolutely necessary.

Reply:
Yes this is for the subtle button. This PR is already merged but if you would like I can create a new PR to remove `!important`

Reply:
I'd probably clean it up, but it's not a big deal.  Mostly worried about it being repeated.  Our goal should be to never use it, but I know we need to use it from time to time.

Reply:
Ok, it wouldn't be much work for me to create a new PR. I'll do that now. Thanks for bringing it up.